core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy

```diff
@@ -21,7 +21,7 @@ l.layout(permission:app.SYSTEM_READ, title:my.displayName) {
             l.spinner(text: _("LOADING"))
         }
 
-        f.form(method:"post",name:"config",action:"configure") {
+        f.form(method:"post",name:"config",action:"configure", class: "jenkins-form") {
```

Comment:
What happens to forms in plugins that do not have this `class`?

Reply:
The bottom bar fills the width of the page rather than ends at the width of the form, so nothing major just the shadow is wider than it should be.

Reply:
Why isn't every `f:form` a `jenkins-form`?